src/qibo_cloud_backends/braket_client.py

```diff
@@ -27,12 +29,17 @@ def __init__(self, device=None, verbatim_circuit=False, verbosity=False):
             verbatim_circuit (bool): If `True`, to_braket will wrap the Braket circuit in a verbatim box to run it on the QPU
                                      without any transpilation. Defaults to `False`.
             verbosity (bool): If `True`, the status of the executed task will be displayed. Defaults to `False`.
+            token (str): This parameter is not required for executing circuits on Amazon Braket devices.
+                         It is included for potential future compatibility but should be left as None.
         """
+
         super().__init__()
 
         self.verbatim_circuit = verbatim_circuit
         self.verbosity = verbosity
 
+        if token:
+            pass
```

Comment:
```suggestion
```

thanks, we could even drop this if we decide to ignore it

Reply:
Ok sure!